ui/app/templates/components/generated-item.hbs

```diff
@@ -52,12 +52,12 @@
     <div class="box is-sideless is-fullwidth is-marginless">
       <NamespaceReminder @mode="save" @noun={{itemType}} />
       <MessageError @model={{model}} />
-      <FormFieldGroups @model={{model}} @mode={{mode}} />
+      <FormFieldGroups @model={{model}} @mode={{mode}} @onKeyUp={{action "onKeyUp"}} @validationMessages={{validationMessages}}/>
```

Comment:
Something Chelsea has been good at pointing out to me in my PRs, but whenever we add properties to a component like this, document the properties in the JSDOCS on the component itself.  So on `form-field-groups.js` add the onKeyUp and validationMessages to the documentation snippet below.

```
 * <FormFieldGroups
 *  @model={{mountModel}}
 *  @onChange={{action "onTypeChange"}}
 *  @renderGroup="Method Options"
 * />
 * ```
 *
 * @param [renderGroup=null] {String} - An allow list of groups to include in the render.
 * @param model=null {DS.Model} - Model to be passed down to form-field component. If `fieldGroups` is present on the model then it will be iterated over and groups of `FormField` components will be rendered.
 * @param onChange=null {Func} - Handler that will get set on the `FormField` component.
 *
 */
```